docs/config.md

```diff
@@ -0,0 +1,2448 @@
+---
```

Comment:
Is that required / wanted in this repo? Seems it's quite big and might get out of date anyway, maybe we can generate it dynamically (+ add to .gitignore) and just push to `gh-pages` instead?

Reply:
added to gitignore :)